tests/rados/test_osd_replicated_inconsistency_scenario.py

```diff
@@ -68,8 +71,16 @@ def run(ceph_cluster, **kw):
                 "run the tests"
             )
             return 1
+        mon_obj.set_config(section="osd", name="debug_osd", value="20/20")
```

Comment:
Make setting of the debug levels optional, based on params sent from suite file.